spec/features/register_spec.rb

```diff
@@ -0,0 +1,75 @@
+feature "User register" do  
+
+  scenario 'with valid email and password' do
+    valid_user = build :user
+
+    register_with valid_user
+
+    expect(page).to have_content( "Welcome")
+  end
+
+  # scenario 'with duplicate user_name' do
+  #   first_user  = create :user
+  #   second_user = build  :user
+
+  #   register_with second_user
+
+  #   expect(User.count).to eq 1
+  #   expect(page).to have_content("User name taken")
+  # end
+
+  def register_with user
+    visit '/register'
+
+    fill_in 'email',                 with: user.email, :match => :prefer_exact
+    fill_in 'password',              with: user.password, :match => :prefer_exact
+    fill_in 'name',                  with: user.name, :match => :prefer_exact
+    fill_in 'user_name',             with: user.user_name, :match => :prefer_exact
```

Comment:
Line is too long. [82/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.